packages/lib/src/components/Card/components/CardInput/defaultProps.ts

```diff
@@ -10,10 +10,10 @@ export default {
     hasHolderName: false,
     holderNameRequired: false,
     enableStoreDetails: false,
-    hideCVC: false,
+    // hideCVC: false,
     hasCVC: true,
-    hasStoreDetails: false,
-    storedDetails: null,
+    // hasStoreDetails: false,
+    // storedDetails: null,
```

Comment:
Can we delete these commented default props?

Reply:
yup